docs/_sass/minima/_base.scss

```diff
@@ -288,8 +288,7 @@ table {
     text-align: center;
   }
   .site-header:before {
-    content: "AB-3";
+    content: "Socius";
     font-size: 32px;
   }
-}
-
+}
```

Comment:
Hey BK, I think you need to add another line of whitespace here to pass the 3 checks

Reply:
I see. Thanks for pointing out!